docs/source/install/profile.rst

```diff
@@ -84,6 +84,25 @@ For this profile to work, you need to download the :ref:`PIConGPU source code <i
 .. literalinclude:: profiles/hydra-hzdr/default_picongpu.profile.example
    :language: bash
 
+Summit (ORNL)
+-------------
+
+**System overview:** `link <https://www.olcf.ornl.gov/olcf-resources/compute-systems/summit/>`_
+
+**User guide:** `link <https://www.olcf.ornl.gov/for-users/system-user-guides/summit/>`_
+
+**Production directory:** usually ``$PROJWORK/$proj/`` (`link <https://www.olcf.ornl.gov/for-users/system-user-guides/summit/summit-user-guide/#file-systems>`_).
+Note that ``$HOME`` is not mounted on compute nodes as read-only.
```

Comment:
This sentence is a bit confusing. "not ... read-only" could mean *read-and-write* but you mean it is not mounted at all. Just remove ~`as read-only`~.

Reply:
```suggestion
Note that ``$HOME`` is mounted on compute nodes as read-only.
```

Reply:
@ax3l While I was confused by the grammar I overlooked that the statement was actually wrong. Great catch. It **is** read-only according the the [documentation](https://www.olcf.ornl.gov/for-users/system-user-guides/summit/summit-user-guide/#storage-overview).